examples/energy-management-app/energy-management-common/energy-management-app.matter

```diff
@@ -2210,6 +2271,56 @@ cluster EnergyEvseMode = 157 {
   command ChangeToMode(ChangeToModeRequest): ChangeToModeResponse = 0;
 }
 
+/** Attributes and commands for selecting a mode from a list of supported options. */
+cluster WaterHeaterMode = 158 {
+  revision 1;
+
+  enum ModeTag : enum16 {
+    kOff = 16384;
+    kManual = 16385;
+    kTimed = 16386;
+  }
+
+  bitmap Feature : bitmap32 {
+    kOnOff = 0x1;
+  }
+
+  struct ModeTagStruct {
+    optional vendor_id mfgCode = 0;
+    enum16 value = 1;
+  }
+
+  struct ModeOptionStruct {
+    char_string<64> label = 0;
+    int8u mode = 1;
+    ModeTagStruct modeTags[] = 2;
+  }
+
+  readonly attribute ModeOptionStruct supportedModes[] = 0;
+  readonly attribute int8u currentMode = 1;
+  attribute optional nullable int8u startUpMode = 2;
```

Comment:
I think startupMode and OnMode should be turned off now - spec change coming through which disallows these.

Reply:
Spec already updated and no XML change needed.